perun-openapi/openapi.yml

```diff
@@ -4737,6 +4737,296 @@ paths:
           $ref: '#/components/responses/ExceptionResponse'
 
+  /json/groupsManager/getAllGroups:
+    get:
+      tags:
+        - GroupsManager
+      operationId: getAllGroups
+      summary: Returns all groups in a VO.
+      parameters:
+        - $ref: '#/components/parameters/voId'
+      responses:
+        '200':
+          $ref: '#/components/responses/ListOfGroupsResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /json/groupsManager/getAllRichSubGroupsWithAttributesByNames:
+    post:
+      tags:
+        - GroupsManager
+      operationId: getAllRichSubGroupsWithGroupAttributesByNames
+      summary: Returns all AllRichSubGroups from parent group containing selected attributes (all level subgroups).
+      responses:
+        '200':
+          $ref: '#/components/responses/ListOfRichGroupsResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              title: GetAllRichSubGroupsWithAttributesByNames
+              description: "input to getAllRichSubGroupsWithAttributesByNames"
+              type: object
+              required:
+                - group
+              properties:
+                group: {type: integer, description: "group id", nullable: false}
+                attrNames:
+                  type: array
+                  items:
+                    type: string
+                  default: []
+
+  /json/groupsManager/createGroup/v-n-d:
+      post:
+        tags:
+          - GroupsManager
+        operationId: createGroupWithVoNameDescription
+        description: Creates a new group in the specific VO.
+        responses:
+          '200':
+            $ref: '#/components/responses/GroupResponse'
+          default:
+            $ref: '#/components/responses/ExceptionResponse'
+        requestBody:
+          required: true
+          content:
+            application/json:
+              schema:
+                title: CreateGroup
+                description: "input to createGroup"
+                type: object
+                required:
+                  - vo
+                  - name
+                  - description
+                properties:
+                  vo: {type: integer, description: "vo id", nullable: false}
+                  name: {type: string}
+                  description: {type: string}
+
+  /json/groupsManager/createGroup/pg-n-d:
+    post:
+      tags:
+        - GroupsManager
+      operationId: createGroupWithParentGroupNameDescription
+      description: Creates a subgroup of a group.
+      responses:
+        '200':
+          $ref: '#/components/responses/GroupResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              title: CreateGroup
```

Comment:
I think that it is not possible to use the same "title" for two different objects. 

Also I would prefer to use "/urlinjsonout/"way of calling with URL parameters instead of JSON object request body.